gettsim/tests/test_interface.py

```diff
@@ -293,3 +295,33 @@ def arbeitsl_geld_2_m_double(arbeitsl_geld_2_m):
     np.testing.assert_array_almost_equal(
         out["arbeitsl_geld_2_m_double_hh"], expected_res
     )
+
+
+def test_convert_series_to_internal_types():
```

Comment:
Would be great if you can use `@pytest.mark.parametrize` (see some examples in the file above) such that each input/series is run as a separate test.